File: Easy/Java/squaresOfASortedArray.java
// Solution: Loop through the array and get the absolute value. Then square.
// Runtime: O(n)

class Solution {
    public int[] sortedSquares(int[] A) {

        for (int i = 0; i < A.length; i++) {
            int val = Math.abs(A[i]);
            A[i] = val * val;
        }

        Arrays.sort(A);

        return A;
    }
}

File: Easy/JavaScript/squaresOfASortedArray.js
/**
 * @param {number[]} A
 * @return {number[]}
 */
var sortedSquares = function(A) {
    
    let ans = new Array(A.length);
    
    for (let i = 0; i < A.length; i++) {
        ans[i] = A[i] * A[i];
    }
    
    return ans.sort((a,b) => a-b);
};
